contracts/beacon/Voucher.sol

```diff
@@ -295,6 +290,23 @@ contract Voucher is OwnableUpgradeable, IVoucher {
         return $._sponsoredAmounts[dealId];
     }
 
+    /**
+     * Retrieve the address of the voucher hub associated with the voucher.
+     * @return voucherHubAddress The address of the voucher hub.
+     */
+    function getVoucherHub() public view returns (address) {
```

Comment:
I wonder if it is not cheaper to set it as `external` and call `$.hub` where we need it (1 call only) (?)

Reply:
+ 12 gas on `sponsor/MatchOrders/Boost`

Reply:
Moving from `public` to `external` increase 12 in this context?

Reply:
Yes

Reply:
Okay thanks!